mods/cnc/maps/nod04b/rules.yaml

```diff
@@ -5,7 +5,7 @@ World:
 		StartingMusic: valkyrie
 		VictoryMusic: nod_win1
 	MissionData:
-		Briefing: A small village friendly to our cause has been increasingly harassed by GDI, and the Brotherhood wishes you to assist them in their efforts.\n\nSeek out the enemy village and destroy it. The event will be disguised as a GDI attack.
+		Briefing: A small village friendly to our cause has faced increasing harassment from GDI. The Brotherhood requires you to assist them in their efforts.\n\nSeek out the village and destroy it. The operation will be disguised as a GDI attack.
```

Comment:
"wishes" here is also important flavour that relates to the way that Seth "requests" you to do things.